### PR TITLE
better deal with loading files and add eslint

### LINKED ISSUE #26: Feedback on the wording of exercices
`camp2_exercises/02_programming_fundamentals/03_algorithms_day_1/`
- `03_play_with_conditional_statement `
  - `01_can_i_drink_alcohol `
    - [ ] say explicitly that we are expecting a boolean
  - `02_can_i_rent_a_car `
    - [ ] say explicitly that we should use the variables assigned at the top of the file
    - [ ] say explicitly that the necessary licenses are only `B` **AND** `B1`
- `04_play_with_for_loop`
  - `01_print_numbers `
    - [ ] say explicitly that we should use a loop
    - [ ] explain `stdout`

### PR DESCRIPTION
**Pull request by Fenntasy**
closes #26

### CODE REVIEW

File: 02_programming_fundamentals/03_algorithms_day_1/.eslintrc.js [hunk 1]
```diff
@@ -8,7 +8,10 @@ module.exports = {
     },
     "extends": "eslint:recommended",
     "rules": {
-        "indent": ["error", 2], // 2 spaces indentation
+        "indent": ["error", 2, { // 2 spaces indentation
+          "SwitchCase": 1, // indentation inside switch
+          "ObjectExpression": 1, // indentation inside objects
```
Comment (arnaudmorisset) [inline 02_programming_fundamentals/03_algorithms_day_1/.eslintrc.js]: Oh 😲 only one for switch and objects?

File: 02_programming_fundamentals/03_algorithms_day_1/03_play_with_conditional_statement/.__tests__/01_can_i_drink_alcohol.test.js [hunk 2]
```diff
@@ -1,81 +1,79 @@
...
+    const francis = eval(code + "; francis;");
 
-  expect(francis.age).toBe(17);
-  expect(francis.name).toBe("Francis");
+    expect(francis.age).toBe(17);
+    expect(francis.name).toBe("Francis");
+  });
 });
 
 describe("Frieda and alchohol", () => {
```
Comment (arnaudmorisset) [inline 02_programming_fundamentals/03_algorithms_day_1/03_play_with_conditional_statement/.__tests__/01_can_i_drink_alcohol.test.js]:
```diff
- alchohol
+ alcohol
```

File: 02_programming_fundamentals/03_algorithms_day_1/03_play_with_conditional_statement/.__tests__/01_can_i_drink_alcohol.test.js [hunk 3]
```diff
@@ -1,81 +1,79 @@
...
+        changedStudentCode + "; canFriedaDrinkAlcohol;"
+      );
 
-    expect(canFriedaDrinkAlcohol).toBe(false);
+      expect(canFriedaDrinkAlcohol).toBe(false);
+    });
   });
 });
 
 describe("Francis and alchohol", () => {
```
Comment (arnaudmorisset) [inline 02_programming_fundamentals/03_algorithms_day_1/03_play_with_conditional_statement/.__tests__/01_can_i_drink_alcohol.test.js]: Same as above. 🙂

File: 02_programming_fundamentals/03_algorithms_day_1/04_play_with_for_loop/.__tests__/05_descending.test.js [hunk 4]
```diff
@@ -1,42 +1,46 @@
...
+  studentCode = readcode(path.resolve(__dirname, "../05_descending.js"));
+  return studentCode;
 });
 
 test("print numbers from zero to nine", () => {
-  numbers = [];
-  _consolelog = console.log;
-  console.log = thing => numbers.push(thing);
+  return studentCode.then(code => {
+    numbers = [];
```
Comment (arnaudmorisset) [inline 02_programming_fundamentals/03_algorithms_day_1/04_play_with_for_loop/.__tests__/05_descending.test.js]: Where numbers is initially declared? 🤔

File: 02_programming_fundamentals/03_algorithms_day_1/05_play_with_for_loop_bonus/.__tests__/04_inverted_right_triangle.test.js [hunk 5]
```diff
@@ -1,35 +1,27 @@
...
 });
 
 test("display an inverted right triangle", () => {
-  output = [];
-  _consolelog = console.log;
-  console.log = thing => output.push(thing);
+  return studentCode.then(code => {
+    output = [];
+    _consolelog = console.log;
+    console.log = thing => output.push(thing);
```
Comment (arnaudmorisset) [inline 02_programming_fundamentals/03_algorithms_day_1/05_play_with_for_loop_bonus/.__tests__/04_inverted_right_triangle.test.js]: Is this relevant to make a `collectOutput` function? 🤔

File: 02_programming_fundamentals/03_algorithms_day_1/.eslintrc.js (hunk 1 above)
Comment (Fenntasy) [reply 02_programming_fundamentals/03_algorithms_day_1/.eslintrc.js]: Not, it's just the way of activating the option because eslint is cool and all but the configuration is still way behind its time.

File: 02_programming_fundamentals/03_algorithms_day_1/05_play_with_for_loop_bonus/.__tests__/04_inverted_right_triangle.test.js (hunk 5 above)
Comment (Fenntasy) [reply 02_programming_fundamentals/03_algorithms_day_1/05_play_with_for_loop_bonus/.__tests__/04_inverted_right_triangle.test.js]: I think there is a way of doing that cleanly with Jest but I'll do another PR
